src/engraving/dom/chord.cpp

```diff
@@ -2684,7 +2684,7 @@ Chord* Chord::nextTiedChord(bool backwards, bool sameSize)
 bool Chord::containsTieEnd() const
 {
     for (const Note* note : m_notes) {
-        if (note->tieBack() && !note->tieFor()) {
+        if ((note->tieBack() && note->tieBack()->playSpanner()) && !(note->tieFor() && note->tieFor()->playSpanner())) {
```

Comment:
I think this is questionable, because it assumes that this method is playback-related, which happens to be true, but isn't reflected by the name.
I see a few options:
1. Rename the method to reflect that it is playback-related
2. Add a parameter to it to specify whether it should be playback-related or not
3. Remove it from the `Chord` class, and put it as a static method in the (only) file where it is called

I have a slight preference for option 3.